app/main.py

```diff
@@ -1,8 +1,52 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
+        if 1 <= comfort_class <= 7:
+            self.comfort_class = comfort_class
+        if 1 <= clean_mark <= 10:
+            self.clean_mark = clean_mark
```

Comment:
what if I set 11 for each field? It will not set any attribute, thus class instance won't have them and python will raise an error that class attribute doesn't have these attributes

Reply:
Yes, it's. You can do whatever you want in class methods :)